src/main/java/net/oauth/jsontoken/JsonTokenParser.java

```diff
@@ -103,12 +105,18 @@ public JsonToken verifyAndDeserialize(String tokenString) throws SignatureExcept
    */
   private List<Verifier> provideVerifiers(JsonToken jsonToken) {
     Preconditions.checkNotNull(verifierProviders);
-    List<Verifier> verifiers = verifierProviders
-        .getVerifierProvider(jsonToken.getSignatureAlgorithm())
-        .findVerifier(jsonToken.getIssuer(), jsonToken.getKeyId());
+    SignatureAlgorithm signatureAlgorithm = jsonToken.getSignatureAlgorithm();
+    VerifierProvider provider = verifierProviders.getVerifierProvider(signatureAlgorithm);
```

Comment:
Can you add the @Nullable annotation to the getVerifierProvider method in VerifierProviders?

Reply:
Added